### ISSUE TITLE
Is it possible to override the normalizePath from the config?

### ISSUE THREAD

**Issue by ayashjorden** [jochen-schweizer/express-prom-bundle#15]
Hello,
In the [documentation](https://github.com/jochen-schweizer/express-prom-bundle#more-details-on-includepath-option) it says that one needs to override the original `normalizePath` function to add custom logic.

Is it possible to do so also by specifying a `normalizePath` key in the config passed to the `promBundle` function call?
for example: (my config, in the code i call `promBundle(config.get("prometheus"))`)
```
    prometheus:
        includeMethod: true
        includePath: true
        promClient:
            collectDefaultMetrics:
                timeout: 1000
         normalizePath:
             '\/auth\/token\/.*$': '/auth/token/*' => this will get magically parsed and added to the paths-to-normalize
```

Thank you,
Yarden

**Comment by disjunction** [comment]
this sounds like a good idea. I'll see if i have time to code that. I would change the format though to:

```
...
normalizePath:
  - ^\/auth\/token\/(.*)$
  - ^\/people\/(.*)\/account\/(.*)$
```
each bracket would be converted to `#val`

**Comment by disjunction** [comment]
... on the second thought your suggestion is more flexible. I've implemented in v 4.1.0 (just published on npm). It's different from your suggestion in terms how the regexps are defined: it's all based on arrays instead of object with regex as a key.

See commit 5e0cd75

Example:
```javascript
app.use(promBundle({
  normalizePath: [
     ['^/customer/.*', '/customer/#name'],
     ['^.*/order-list', '/#name/order-list']
  ]
}));
```
